core/src/main/java/de/danoeh/antennapod/core/widget/WidgetUpdater.java

```diff
@@ -65,8 +68,10 @@ public WidgetState(PlayerStatus status) {
      */
     public static void updateWidget(Context context, WidgetState widgetState) {
         if (!PlayerWidget.isEnabled(context) || widgetState == null) {
+            Log.d(TAG, "updateWidget: enteredstatement");
             return;
         }
+        Log.d(TAG, "updateWidget: enteredmain");
```

Comment:
Please remove the extra debug statements

Reply:
@tonytamsf Sure